### DISCUSSION TITLE
Contribution procedure

### DISCUSSION THREAD

**Discussion by mislavmatijevic** [foivz/pop-app-backend#7] [Ideas]
In order to contribute to the project:

- First select or add an issue. It needs to be a complete component with a DoD written in it.

- Assign self to issue.

- Locally use `git flow feature start feature/#{issueId}_name-of-the-issue`.

- _... commits go here ..._

- Locally use `git flow feature finish feature/#{issueId}_name-of-the-issue`.

- Locally use `git flow feature publish feature/#{issueId}_name-of-the-issue` to push to remote.

- Open PR and add a reviewer.

- Reviewer closes PR and deletes branch, you close the issue with a reference to the PR in the closing comment.

Done by following the: https://danielkummer.github.io/git-flow-cheatsheet/#features
@antoniokunstek Could you please just double-check? Ty ❤️

**Comment by antoniokunstek** [comment]
Dear Mislav (@mislavmatijevic),

everything looks right.
I am sending you an additional link that I used when working with git flow, but I believe you have already encountered it.

Best regards!
Antonio Kunštek

Additional link: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

**Comment by mislavmatijevic** [comment]
# Problem

As far as PRs are concerned, I noticed this:
![slika](https://github.com/foivz/pop-app-backend/assets/11699688/406f4766-7d50-4118-a83b-a728869913fa)

I'd prefer it to be like this:
![slika](https://github.com/foivz/pop-app-backend/assets/11699688/c4c50e13-906e-4038-92b5-eb5090bcf059)

## Reasoning
You can see the author from the PR page.

### 🔴 Bad 
![slika](https://github.com/foivz/pop-app-backend/assets/11699688/0e281a18-5457-47ca-82e3-1af8cf994ccf)

### ✔️ Good 
![slika](https://github.com/foivz/pop-app-backend/assets/11699688/2d06230a-7fb5-4a4f-a0d8-cfa4e24518b1)

**Reply by antoniokunstek** [reply]
Sorry. My bad. I notice it when you changed the Assignee on PR. Thanks for info, tho! 😋
